contracts/ExitHandler.sol

```diff
@@ -22,30 +22,29 @@ contract ExitHandler is IExitHandler, DepositHandler {
 
   using PriorityQueue for PriorityQueue.Token;
 
+  /**
+    - tokenData — (optional) NST data
+   */
   event ExitStarted(
-    bytes32 indexed txHash,
-    uint8 indexed outIndex,
-    uint256 indexed color,
-    address exitor,
-    uint256 amount
-  );
-
-  event ExitStartedV2(
     bytes32 indexed txHash,
     uint8 indexed outIndex,
     uint256 indexed color,
     address exitor,
     uint256 amount,
-    bytes32 data
+    bytes32 tokenData
   );
 
+  /**
+    - tokenData — (optional) NST data
+   */
   struct Exit {
     uint256 amount;
     uint16 color;
     address owner;
     bool finalized;
     uint32 priorityTimestamp;
     uint256 stake;
```

Comment:
if you make stake a uint32 (for example multiple of 0.1 ETH), then you save one storage slot.

Reply:
out of scope, but noted here: https://github.com/leapdao/leap-contracts/issues/207